tools/istio-docker.mk

```diff
@@ -22,7 +22,7 @@
 docker: build test-bins docker.all
 
 DOCKER_TARGETS:=docker.pilot docker.proxy_debug docker.proxytproxy docker.proxyv2 docker.app docker.test_policybackend \
-	docker.proxy_init docker.mixer docker.citadel docker.galley docker.sidecar_injector
+	docker.proxy_init docker.mixer docker.citadel docker.galley docker.sidecar_injector docker.kubectl
```

Comment:
I don't know if we need to build this on each presubmit - but since it's now fast it is ok.